pandablocks/commands.py

```diff
@@ -293,6 +294,30 @@ def execute(self) -> ExchangeGenerator[None]:
         assert ex.line == "OK"
 
 
+@dataclass
+class Append(Command[None]):
+    """Append the value of a table field.
+
+    Args:
+        field: The field, attribute, or star command to append
+        value: The value, list of strings, to append
+
+    For example::
+
+        Put("SEQ1.TABLE", ["1048576", "0", "1000", "1000"])
+    """
+
+    field: str
+    value: Union[List[str]] = ["",]
```

Comment:
As this is only for putting to tables, which are multiline, we can simplify the type and make it required...

```suggestion
    value: List[str]
```